workflow.src

```diff
@@ -0,0 +1,121 @@
+      </p>
+      <pre><code> git push --set-upstream origin my-wonderful-edits-for-87</code></pre>
+    </li>
+    <li>
+      <p>
+        Using your favorite browser, navigate to the repo on GitHub, e.g.
+      </p>
+      <pre><code>https://github.com/tobie/specs-on-github/</code></pre>
+    </li>
+    <li>
+      <p>
+        Once your anthenticated on the repo page, a new button should appear on the page to create a new
+        pull request (<code>Compare &amp; pull request</code>). Go ahead and punch it.
+      </p>
+      <p><img src="img/create-a-pr.png" alt="Compare &amp; pull request" /></p>
+    </li>
+    <li>
+      <p>
+        Make sure you like the title of your pull request, add a proper description, and use
+        <a href="spec-labels.html">proper GitHub labels</a>, add one or more reviewers so they get pinged.
+        Once you're satisfied, create the pull request. Note that you can update all of those information
+        after that if needed.
+      </p>
+    </li>
+    <li>
+     <p>
+       ALL normative spec changes are generally expected to have a corresponding pull request in
+       <a href="https://github.com/w3c/web-platform-tests/">web-platforms-tests</a>,
+       either in the form of new tests or modifications to existing tests, or must include the rationale for why test
+       updates are not required for the proposed update.
+     </p>
+     <p>
+       Typically, both pull requests (spec updates and tests) will be merged at the same time. If a pull request
+       for the specification is approved but the other needs more work, add the
+       '<a href="https://w3c.github.io/spec-labels.html">needs tests</a>' label or,
+       in web-platform-tests, the
+       '<a href="https://github.com/w3c/web-platform-tests/issues?utf8=%E2%9C%93&q=label%3Astatus%3Aneeds-spec-decision%20">status:needs-spec-decision</a>'
+       label. Note that a test change that contradicts
+       the specification should not be merged before the corresponding specification change.
+     </p>
+     <p>
+       If testing is not practical due to <a href="https://github.com/w3c/web-platform-tests/">web-platforms-tests</a> limitations,
+       please explain why and if appropriate file an issue with the
+       '<a href="https://github.com/w3c/web-platform-tests/issues?utf8=%E2%9C%93&q=label%3Atype%3Auntestable%20">type:untestable</a>'
+       label to follow up later.
+     </p>
+    </li>
+    <li>
+      <p>
+        Wait for the needed reviews. All pull requests must have been reviewed by one or more participants of the working groups.
+        They can use approve changes, give you <code>+1</code>, comment, etc.
+      </p>
+    </li>
+    <li>
+      <p>
+        Once the pull request gets enough support, it can be merged (use the <code>Squash and merge</code> to avoid cluttering the commit history). Delete your ad-hoc branch <code>my-wonderful-edits-for-87</code>. You're done.
```

Comment:
I don't think "squash and merge" is necessarily best practice - I don't think we should recommend it

Reply:
It depends... we don't want a commit history that looks like:

* closes #someNumber 
* fixed typo
* another fixed typo
* tipo is typographical typo. 
* Review stuff

We want (squashed):

* Editorial: fixed typo (closes #someNumber)
 
And maybe additional stuff...

Reply:
To be clear: it depends on the Editor understanding git... what we don't want is for multiple people to be working on a change, and for one editor to squash and then force push and break all the things... which is why "squash and merge" is nice.

Reply:
yes - but the document also suggests making several distinct commits for semantics, which get lost in squashing. I don't think there is an always-right approach - maybe it should document what the options are and when they're best applied.

Reply:
Agree - it's might be good to explain the options. But I think we all agree that we don't want to end up with a commit history like the one I mentioned above (i.e., "oops" and "fixed typo" etc.).

Reply:
As a first approach, it might be better to stay silent on that part then and let folks figure this one out?